sensor/kubernetes/listener/resources/deployment_store_test.go

```diff
@@ -109,6 +110,11 @@ func (s *deploymentStoreSuite) Test_FindDeploymentIDsWithServiceAccount() {
 }
 
 func (s *deploymentStoreSuite) Test_BuildDeployments_CachedDependencies() {
+	s.T().Setenv(features.SensorDeploymentBuildOptimization.EnvVar(), "true")
+	if !features.SensorDeploymentBuildOptimization.Enabled() {
```

Comment:
I don't think this is needed anymore. Now FF can be enabled in production (i.e. release tests) so as long as you are setting the env above this check isn't necessary.

Reply:
You're right! Old habits 😆